document-store/src/main/java/org/hypertrace/core/documentstore/mongo/parser/MongoFunctionExpressionParser.java

```diff
@@ -0,0 +1,31 @@
+package org.hypertrace.core.documentstore.mongo.parser;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.hypertrace.core.documentstore.expression.impl.FunctionExpression;
+import org.hypertrace.core.documentstore.parser.SelectingExpressionParser;
+
+public class MongoFunctionExpressionParser {
+
+  static Map<String, Object> parse(FunctionExpression expression) {
+    int numArgs = expression.getOperands().size();
+
+    if (numArgs == 0) {
```

Comment:
Probably these validations are no longer required with Hibernate validations taking place while building the query.